src/isPalindrome.js

```diff
@@ -0,0 +1,11 @@
+export default function isPalindrome(str) {
```

Comment:
```js
export default function isPalindrome(str) {
  const formattedStr = str.toLowerCase().replace(/[^a-z|0-9]/g, "")
  return formattedStr == reversedString(formattedStr)
}

function reversedString(formattedStr) {
  return formattedStr === "" 
    ? ""
    : reversedString(formattedStr.substr(1)) + formattedStr.charAt(0)
}
```

if you define `reversedString` outside of `isPalindrome` than you do not need to define it every time `isPalindrome` is called. Should be faster and easier to understand